packages/web/utils/menuLinks.ts

```diff
@@ -0,0 +1,157 @@
+// in utils
+
+export interface MenuLink {
```

Comment:
This interface is off. It nearly describes the nested objects, but is missing a couple fields. It is used as the type for the `MenuSectionLinks` which should just be something like:

```typescript
export interface MenuLinkSet {
  label: string;
  menuItems: MenuLink[];
}
```